src/excmds.ts

```diff
@@ -1881,23 +1882,22 @@ export async function tabopen(...addressarr: string[]) {
         return args
     }
 
-    let url: string
-    let address = (await argParse(addressarr)).join(" ")
+    let query = await argParse(addressarr)
 
-    if (address == "") address = config.get("newtab")
+    let address = query.join(" ")
     if (!ABOUT_WHITELIST.includes(address) && address.match(/^(about|file):.*/)) {
         return nativeopen(address)
-    } else if (address != "") url = forceURI(address)
```

Comment:
`forceURI` did quite a bit of stuff, and I'm not convinced that you've replicated it faithfully.

For example, `open search test` now searches Google for `search test`, rather than just `test` as expected.